server/src/main/java/io/unitycatalog/server/persist/dao/CatalogInfoDAO.java

```diff
@@ -17,7 +17,7 @@
 @NoArgsConstructor
 public class CatalogInfoDAO {
   @Id
-  @Column(name = "id", columnDefinition = "BINARY(16)")
+  @Column(name = "id")
```

Comment:
Do we make these changes because postgre doesn't support them? How do we know what column type are they mapped to in each database backend?

Reply:
Yes, we made these changes because columnDefinition is somewhat dialect specific and postgre does not recognize all. Hibernate will try its best to map to the correct type for each backend database.